specs/pipeline/edit/GeneralSettings.spec

```diff
@@ -0,0 +1,25 @@
+// --GO-LICENSE-START--
```

Comment:
Please get rid of "GO-LICENSE-START" —

This was added in the old code because of a script that used to assert the presence of the license header. Which was never being called anywhere. This was gotten rid of some time ago, but we continue to add this unnecessary header.